docs/features/logforwarding/outputs/otlp-forwarding.adoc

```diff
@@ -0,0 +1,139 @@
+          token:
+            from: serviceAccount  <5>
+  pipelines:
+   - name: my-pipeline
+     inputRefs:
+     - application
+     - infrastructure
+     outputRefs:
+     - otel-collector
+  serviceAccount:
+    name: logger-admin
+----
+. Output `type` is '*otlp*'
+. `tls` includes the standard certificate configuration or specify `insecureSkipVerify`: '*true*'
+. `otlp` `url` specify a valid host:port of the otel receiver and *MUST* terminate with "*/v1/logs*"
+. `otlp` `tuning` `compression` default value is "*gzip*". Specify "*none*" to turn compression off.
+. `otlp` `authentication` is optional and specifies a `token` `from` and a value of "*serviceAccount*"
+.. The token can also be read from a secret
+.. Also available with `username` and `password` authentication spec (refer to HTTP Auth Specification for full scope)
+
+
+.TLS InsecureSkipVerify
+[NOTE]
+This option is *NOT* recommended for production configurations. If true, the client will be configured to skip validating server certificates.
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L244[TLS Specification] for _observability.openshift.io/v1_
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L203[HTTP Auth Specification] for _observability.openshift.io/v1_
+
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L812[Tuning Specification] for _observability.openshift.io/v1_
+
+
+== Data Model
+
+=== Semantic Convention
+The Semantic Conventions in OpenTelemetry define a *Resource* is an immutable representation of the entity producing telemetry as *Attributes*.
+
+For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
+
+All of these *Attributes* are included in the *Resource* object.  This grouping and reducing of common attributes is a powerful tool when sending logs as telemetry data.
+
+.OTEL References
+* https://opentelemetry.io/docs/specs/semconv/[Semantic Conventions]
+* https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+* https://opentelemetry.io/docs/specs/semconv/general/logs/[General Logs Attributes]
+
+=== Data Mapping
+This document is not a complete spec and is mainly used as a guide.
+
+==== JSON Encoding
+OpenShift Logging uses Vector Remap Language https://vector.dev/docs/reference/vrl/[(VRL)] to map and enrich the logging data in order to the follow the semantic conventions of OpenTelemetry and its https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+
+https://github.com/open-telemetry/opentelemetry-proto/blob/v1.3.1/examples/logs.json[Example JSON Encoded Logs] provide a visual representation of the data structure
+
+[%header,format=csv]
+|===
+Viaq,OTEL
+openshift.cluster_id,       resource.attributes.cluster.id
```

Comment:
Here we define the OTEL level where these attributes will be found in the final message.  Given our discussions, I am led to believe the "level" does not matter when we query this information.  I would consider excluding that part of it since it is irrelevant from query point of view

Reply:
I was using this as a mapping guide as well.   I will remove "resource" and "logRecords" for this doc.